articles/building-a-simple-cryptocurrency-blockchain/index.md

```diff
@@ -0,0 +1,187 @@
+## Building Simple cryptocurrency system using Node.js
+
+In this tutorial, we are going to demonstrate how to build a simple cryptocurrency system, called thecoin.
+It is a simple implementation of the of decentralized digital money, based on blockchain technology. 
```

Comment:
The introduction could have been better. Have a look at the intros of other Section articles, to get an idea.

You could introduce by stating about cryptocurrencies, then give 1 - 2 lines explanation about what blockchain is, and then talk about "thecoin".

The introduction and the conclusion are the most important parts of a tutorial. So, you must make sure that the reader must gain interest by reading them.